README.md

```diff
@@ -1,5 +1,7 @@
 # @ui5/manifest
 
+[![REUSE status](https://api.reuse.software/badge/github.com/fsfe/reuse-tool)](https://api.reuse.software/info/github.com/fsfe/reuse-tool)
```

Comment:
Shouldn't this point to your project (https://api.reuse.software/info/github.com/SAP/ui5-manifest) rather than the fsfe/reuse-tool?